src/__tests__/helpers/tokens.ts

```diff
@@ -0,0 +1,14 @@
+import { BigNumberish, Contract, Overrides } from "ethers";
+import { ERC721 } from "../../typechain/solmate/src/tokens/ERC721.sol/ERC721";
+import abiIERC721 from "../../abis/IERC721.json";
+import { Signer } from "../../types";
+
+export const balanceOf = async (signer: Signer, contractAddress: string, owner?: string, overrides?: Overrides) => {
```

Comment:
do you still need a signer for readonly functions?

Reply:
No but because it's a helper for the test, it makes it easier (I can ignore the address and pull it from the signer)